src/components/packages/Package.js

```diff
@@ -370,7 +379,7 @@ class Package extends Component {
                                     key={feature.id}>
                                     <Icon icon={['fas', 'check']} />
                                     <span className="fs-feature-title">{featureTitle}</span>
-                                    {Helper.isNonEmptyString(feature.description) && <Tooltip><Fragment>{feature.description}</Fragment></Tooltip>}
+                                    {Helper.isNonEmptyString(feature.description) && <Tooltip direction={this.getTooltipDirection()}><Fragment>{feature.description}</Fragment></Tooltip>}
```

Comment:
Please remove the `<Fragment>` and kindly refactor to ternary as mentioned in my earlier comment.